packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -0,0 +1,82 @@
+import { ax, makeStyles } from '@fluentui/react-make-styles';
+import { PresenceBadgeState } from './PresenceBadge.types';
+import { useRootStyles as useBadgeStyles } from '../../Badge';
+
+export const usePresenceBadgeRootStyles = makeStyles<PresenceBadgeState>([
+  [
+    null,
+    {
+      padding: 0,
+    },
+  ],
+  [
+    s => s.status === 'busy',
+    theme => ({
+      backgroundColor: theme.global.palette.cranberry.primary,
+      borderColor: theme.global.palette.cranberry.primary,
+    }),
+  ],
+  [
+    s => s.status === 'away',
+    theme => ({
+      backgroundColor: theme.global.palette.yellow.primary,
+      borderColor: theme.global.palette.yellow.primary,
+    }),
+  ],
+  [
+    s => s.status === 'available',
+    theme => ({
+      backgroundColor: theme.global.palette.lightGreen.primary,
+      borderColor: theme.global.palette.lightGreen.primary,
+    }),
+  ],
+  [
+    s => s.status === 'offline',
+    theme => ({
+      backgroundColor: 'transparent',
+      color: theme.global.palette.grey[38],
+      borderColor: theme.global.palette.grey[38],
+    }),
+  ],
+  [
+    s => s.status === 'outOfOffice',
+    theme => ({
+      backgroundColor: 'transparent',
```

Comment:
Looking at the visual spec, I think this is supposed to be
```suggestion
      backgroundColor: theme.alias.color.neutral.neutralBackground1,
```